tests/devtools/useGoToAtomsSnapshot.test.tsx

```diff
@@ -20,7 +20,7 @@ it('[DEV-ONLY] useGotoAtomsSnapshot should modify atoms snapshot', async () => {
   }
 
   const UpdateSnapshot = () => {
-    const snapshot = useAtomsSnapshot()
+    const [snapshot] = useAtomsSnapshot()
```

Comment:
So, I mean we don't change this, but change useGotoAtomsSnapshot(). Luckily, for this one, we can make it backward compatible with `Array.isArray()`.